libdevcore/CommonData.h

```diff
@@ -99,6 +119,12 @@ inline std::set<T> operator+(std::set<T>&& _a, U&& _b)
 	return ret;
 }
 
+template <class T, class U>
+T initFrom(U const& _from)
```

Comment:
Sounds like a useful function, but maybe not in the global namespace? Also it should provide a move variant and maybe a different name...

Reply:
Maybe `convertContainer`?

Reply:
Moved to `dev` namespace and renamed